code/modules/mob/living/carbon/human/species/station/protean_vr/protean_species.dm

```diff
@@ -30,34 +31,41 @@
 	blood_volume =	0
 	min_age =		18
 	max_age =		200
-	brute_mod =		0.2 //Brute isn't very effective, they're made of dust
-	burn_mod =		2.0 //Burn, however, is
+	brute_mod =		0.40 //Brute isn't very effective, they're made of dust, bump down to 0.2 after revive coded
+	burn_mod =		1.5 //Burn, is more effective due to Graphene conduits. Until revive is coded in, then may bump proteans back to original 2 value
 	oxy_mod =		0
 
 	cold_level_1 = 280 //Default 260 - Lower is better
 	cold_level_2 = 220 //Default 200
 	cold_level_3 = 130 //Default 120
 
-	heat_level_1 = 320 //Default 360
-	heat_level_2 = 370 //Default 400
-	heat_level_3 = 600 //Default 1000
+	heat_level_1 = 552 //Default 360
+	heat_level_2 = 553 //Default 400
+	heat_level_3 = 570 //Default 1000
```

Comment:
Heat resistant synths?
Do proteans really need this much of a buff?
I'd point out that, compared to the other values these ones are far too close together, but I'm concerned that the fix then would just be to make them even stronger.
Not a fan. 👎

Reply:
See latest comment. Issue with simple mobs so I'll revert this for time being.

The idea behind it was that once past the melting point, proteans die within 2/3 ticks due to the internal structure being lost.